disnake/guild.py

```diff
@@ -3404,3 +3403,54 @@ async def bulk_edit_command_permissions(
             A list of edited permissions of application commands.
         """
         return await self._state.bulk_edit_command_permissions(self.id, permissions)
+
+    async def timeout(
+        self,
+        user: Snowflake = MISSING,
+        *,
+        seconds: Optional[int] = MISSING,
+        reason: Optional[str] = None,
+    ) -> Optional[Member]:
+        """|coro|
+
+        Time outs the member from the guild; until then, the member will not be able to interact with the guild.
+
+        The user must meet the :class:`abc.Snowflake` abc.
```

Comment:
```suggestion
        Times out the member from the guild; until then, the member will not be able to interact with the guild.
```

Reply:
Why is this change?

Reply:
Isn't this exactly what the annotations should be doing? I don't think there's really a need to tell the user twice.

Reply:
I mean, I was just following the same syntax as ban, unban, etc.

Reply:
I'm going to close this, maybe we can discuss this in the server, I have a documentation PR open, so I can commit things there.